wheatley/main.py

```diff
@@ -223,6 +228,9 @@ def main():
 
     args = parser.parse_args()
 
+    if args.wait:
+        print("Deprication warning: `--wait` has been replaced with `--keep-going`!")
```

Comment:
```suggestion
        print("Deprecation warning: `--wait` has been replaced with `--keep-going`!")
```

Reply:
Yup nice my speling is not the best when programming!